common/src/main/java/com/mx/path/core/common/connect/Request.java

```diff
@@ -436,6 +439,19 @@ public final REQ withRetryer(Retryer<RESP> newRetryer) {
     return (REQ) this;
   }
 
+  /**
+   * The preferred way to provide a retryer
+   *
+   * <p>{@link RetryConfigurationNode} can be added to bound configuration POJO and passed directly into this for relevant
+   * configurations.
+   * @param retryConfiguration
+   * @return this
+   */
+  @SuppressWarnings("unchecked")
+  public final REQ withRetryer(RetryConfigurationNode retryConfiguration) {
```

Comment:
What if this was `.withRetryConfiguration(RetryConfiguration retryConfiguration)`

So:
- RetryConfigurationNode -> RetryConfiguration
- withRetryer -> withRetryConfiguration

Reply:
I like it.

Reply:
Renamed.